hack/multi-node/user-data.sample

```diff
@@ -19,15 +19,18 @@ coreos:
           --mount volume=opt-cni-bin,target=/opt/cni/bin \
           --insecure-options=image"
         ExecStartPre=/bin/mkdir -p /opt/cni/bin
-        ExecStartPre=/bin/mkdir -p /etc/kubernetes/manifests
         ExecStartPre=/bin/mkdir -p /etc/kubernetes/cni/net.d
         ExecStartPre=/bin/mkdir -p /etc/kubernetes/checkpoint-secrets
         ExecStartPre=/bin/mkdir -p /etc/kubernetes/inactive-manifests
+        ExecStartPre=/bin/mkdir -p /etc/kubernetes/kubelet/tls
```

Comment:
Default directory for kubeadm is /etc/kubernetes/pki. Does bootkube care to be the same? Just might help people who use both systems but not a real concern for functionality.

Reply:
@rothgar I think that's a good suggestion, but `tls` is used in a few places in bootkube. Mind opening an issue to suggest aligning with kubeadm? We can address in a separate PR.